taskcluster/scripts/fetch-firefoxci-artifacts.py

```diff
@@ -0,0 +1,53 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import json
+import os
+
+import requests
+import taskcluster
+
+from fxci_config_taskgraph.util.integration import FIREFOXCI_ROOT_URL, STAGING_ROOT_URL
+
+if "TASKCLUSTER_PROXY_URL" in os.environ:
+    options = {"rootUrl": os.environ["TASKCLUSTER_PROXY_URL"]}
+else:
+    options = taskcluster.optionsFromEnvironment()
+    assert options["rootUrl"] == STAGING_ROOT_URL
+
+secrets = taskcluster.Secrets(options)
+
+result = secrets.get("project/releng/fxci-config/firefoxci-artifact-client")
+assert isinstance(result, dict)
+
+credentials = result["secret"]
+assert isinstance(credentials, dict)
+
+options = {"rootUrl": FIREFOXCI_ROOT_URL, "credentials": credentials}
+queue = taskcluster.Queue(options)
+
+artifact_dir = os.environ["MOZ_ARTIFACT_DIR"]
+os.chdir(artifact_dir)
+
+task_id = os.environ["FETCH_FIREFOXCI_TASK_ID"]
+artifacts = json.loads(os.environ["FETCH_FIREFOXCI_ARTIFACTS"])
+for artifact in artifacts:
```

Comment:
could we async download the artifacts? (maybe an improvement for another PR)

Reply:
Good idea, though it's probably pretty rare that we'd be fetching more than one artifact from a task anyway. Though thinking about it, we'd need to depend on aiohttp just for this, so I think I'll stick with synchronous for now.